### DISCUSSION TITLE
Feature Request: DC/SS Twin Stick support

### DISCUSSION THREAD

**Discussion by shiro304** [darthcloud/BlueRetro#923] [Ideas]
Hello,

Would it be possible to have a blueretro adaptor detected as a twin stick on the Sega Saturn and Dreamcast and to map a controller or keyboard to them. This would be great for people who want to diy a wireless twin stick for Virtual-on since the official twin sticks are getting rather expensive.

thanks

**Comment by darthcloud** [comment]
I got the Saturn one, so i'll one day add support for it. Not sure when.

Same for DC except I dont have one.

**Comment by daph-proton** [comment]
Came here to show my support for this feature. Using Blueretro to bridge controller gaps like this one would be yet another notch in an already amazing set of features. 

As an aside, thank you for Blueretro and all of your hard work!
